app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split(" ")
+
+    if len(command_list) < 3:
+        return
+
+    cmd = command_list[0]
+    file_name_in = command_list[1]
+    file_name_out = command_list[2]
+
+    if file_name_in == file_name_out or cmd != "cp":
+        return
+
+    with (open(file_name_in, "r") as file_in,
```

Comment:
Imagine situation when file doesn't exist or even more if u don't have permission to read this file